crates/viewer/re_view_time_series/src/point_visualizer_system.rs

```diff
@@ -247,6 +254,17 @@ impl SeriesPointSystem {
                     .map(|index| (index, ()))
             };
 
+            // TODO(#9020): support multiple series.
+            let num_series = 1;
```

Comment:
Is there anything about this that is more complicated than copy/pasting the changes made to `line_visualizer_system`?

Reply:
not really, no. The complication is mostly me not wanting to do a full copy & paste and re-use some stuff better. Maintaining these both in parallel is annoying